docs/guides/building/windows-installer.md

```diff
@@ -0,0 +1,222 @@
+# Windows Installer
+
```

Comment:
@JonasKruckenberg it seems like at the very top of this guide there should be a simple blurb that's just `tauri build` for Windows and noting that you currently need to execute this on Windows since cross-platform compilation isn't currently supported.

Reply:
Chatted with @JonasKruckenberg, going to make a fragment for all OS types that will contain this basic info at the top of each guide